default.nix

```diff
@@ -0,0 +1,12 @@
+let pkgs = import <nixpkgs> {};
+in { stdenv ? pkgs.stdenv, xz ? pkgs.xz }:
+stdenv.mkDerivation {
+  name = "slippc";
+  src = ./.;
+  buildDepends = [ xz ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ./slippc $out/bin/
```

Comment:
Using `bin/` here makes it work with `nix-env -f default.nix` and I think is what Nix expects the `installPhase` to do for executables?